native/src/routes/ImageViewModal.tsx

```diff
@@ -1,37 +1,39 @@
-import React, { ReactElement, useContext } from 'react'
-import { View } from 'react-native'
-import { ImageViewer } from 'react-native-image-zoom-viewer'
+import * as React from 'react'
+import { ReactElement, useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { View, Text } from 'react-native'
 import { ThemeContext } from 'styled-components'
 
 import { ImageViewModalRouteType } from 'api-client'
 
-import { NavigationPropType, RoutePropType } from '../constants/NavigationTypes'
+import PinchableBox from '../components/PinchableBox'
+import { RoutePropType } from '../constants/NavigationTypes'
 
 type PropsType = {
   route: RoutePropType<ImageViewModalRouteType>
-  navigation: NavigationPropType<ImageViewModalRouteType>
 }
 
 const ImageViewModal = ({ route }: PropsType): ReactElement => {
+  const [isError, setError] = useState(false)
   const theme = useContext(ThemeContext)
+  const { t } = useTranslation('error')
+
+  if (isError) {
+    return <Text>{t('imageLoadingFailed')}</Text>
+  }
 
   return (
     <View
       style={{
```

Comment:
Please use a styled component

Reply:
I'm not sure how do do this. The react animated API requires the use of animated styles. 

I think it is cleaner to use the styles directly for animated nodes. I changed it for the View.

Reply:
yes